src/types.ts

```diff
@@ -68,36 +67,39 @@ export interface ConsentHookActions {
 export type ConsentState = CookieTypes;
 
 export interface CookieConsentHookState extends ConsentHookActions {
-  cookies: CookieWrapper;
+  acceptCookies: (cookies: CookieTypes, options?: AcceptCookiesOptions) => void;
+  cookies?: CookieWrapper;
   consent: ConsentState;
 }
 
 export interface StorageConsentHookState extends ConsentHookActions {
+  acceptCookies: (cookies: CookieTypes) => void;
   storage: Storage;
   consent: ConsentState;
 }
 
-export type UseCookieConsentType = (
-  options?: CookieConsentOptions
-) => ConsentHookStateType;
+export type UseCookieConsentHookType = (
+  options?: CookieConsentOptionsCookie
+) => CookieConsentHookState;
+
+export type UseStorageConsentHookType = (
+  options?: CookieConsentOptionsStorage
+) => StorageConsentHookState;
 
-export type ConsentHookStateType =
-  | CookieConsentHookState
-  | StorageConsentHookState;
+export type UseConsentHookType =
+  | UseCookieConsentHookType
+  | UseStorageConsentHookType;
 
 export interface CookieConsentOptionsCookie {
   defaultConsent?: ConsentState;
   consentCookieAttributes?: CookieAttributes;
 }
+
 export interface CookieConsentOptionsStorage {
   defaultConsent?: ConsentState;
   storage?: Storage;
 }
 
-export type CookieConsentOptions =
-  | CookieConsentOptionsCookie
-  | CookieConsentOptionsStorage;
-
 export interface AcceptCookiesOptions {
   allowChangingNecessary?: boolean;
 }
```

Comment:
Wait a second, my bad! The `acceptCookies` function SHOULD have the options, I just mixed them up with the cookie-js options 🤦‍♀️